get-lms-release/action.yml

```diff
@@ -1,31 +1,29 @@
 name: 'Get LMS Version'
-description: 'Retrieves the current dev version of Brightspace from Rally'
+description: 'Retrieves the current active development version of Brightspace'
 inputs:
   RALLY_API_KEY:
     description: 'Token to find active releases in Rally'
     required: true
 outputs:
   LMS_VERSION:
-    description: 'Current dev version of the LMS'
+    description: 'Current active development version of the LMS'
     value: ${{ steps.get-lms-version.outputs.version }}
 
 runs:
   using: composite
   steps:
-    - name: Install Dependencies
-      run: |
-        echo "Installing dependencies..."
-        npm install chalk@5 moment-timezone@0.5 rally@2 --prefix ${{ github.action_path }} --no-save --loglevel error
-      shell: bash
-
     - name: Get LMS Version
       id: get-lms-version
-      uses: actions/github-script@v6
+      run: |
+        VERSION="20.24.3"
+        if [ ${RALLY_API_KEY} != "" ]; then
+          echo "RALLY_API_KEY detected, returning hard-coded release version..."
+        else

```

Comment:
The plan is that when we have the new source of truth working, we'll start switching repos over to NOT pass the Rally API key and instead pass AWS credentials (the new source is in S3). This `else` block will then get filled in for the "new" uses, and the other hard-coded branch will remain in place for a couple months to handle cert-fix and hotfix branches that haven't updated.